codice/ddf-ui
# DDF-UI 365 Adds audit logging to DDF-UI

Description: Adds Audit Logging endpoint.

How to test: 
1. Build DDF-2.26.0 and this PR
2. Upload resources (local and federated), and click on the results.  
3. Under `ddf-2.26.0/data/log/security.log` you should be able to see some logging events about results being clicked
